command/bridge/deploy/deploy.go

```diff
@@ -224,7 +239,9 @@ func deployContracts(
 	externalChainID *big.Int,
 	chainCfg *chain.Chain,
 	initialValidators []*validator.GenesisValidator,
-	cmdCtx context.Context) (*deploymentResultInfo, error) {
+	cmdCtx context.Context,
+	internalGatewayAddress string,
```

Comment:
There is no need for extending deployContracts with gatewayAddress params, just use them from params variable.